pkg/scaffold/v2/main.go

```diff
@@ -145,11 +145,15 @@ func main() {
 
 	ctrl.SetLogger(zap.Logger(true))
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                  scheme,
-		MetricsBindAddress:      metricsAddr,
-		LeaderElection:          true,
-	})
+	op := ctrl.Options{
+		Scheme:             scheme,
+		MetricsBindAddress: metricsAddr,
+		LeaderElection:     true,
+	}
+	if ns := os.Getenv("KUBEBUILDER_LEADER_ELECTION"); len(ns) > 0 {
```

Comment:
Can we take this input as commandline flag ?

Reply:
yeah, I think a flag works fine here.

Reply:
Done